spec/services/catalog/service_plan_json_spec.rb

```diff
@@ -20,6 +20,14 @@
       end
     end
 
+    context "when rendering a service plan from a list of service plans" do
+      let(:params) { {:service_plans => [service_plan]} }
+
+      it "renders the specified service_plan schema" do
+        expect(subject["create_json_schema"]).to eq service_plan.modified
```

Comment:
@lindgrenj6 is this behavior going to change due to https://github.com/RedHatInsights/catalog-api/pull/536

Reply:
No, it shouldn't. Since for a `show` method we aren't passing in the schema it'll just fall back to return the modified if it's there OR the base schema.

There will be a merge conflict though in the spec where we are comparing keys - it'll be easy to fix though, whoever gets their PR in last will have to fix it :P